### DISCUSSION TITLE
Separate skin layer/select for spectate screen

### DISCUSSION THREAD

**Discussion by spacekotek** [ppy/osu#30761] [Ideas]
Opening this because https://ptb.discord.com/channels/188630481301012481/1097318920991559880/1308110695807582309

A playable skin may look pretty bad on spectator screen, and a skin made specifically for spectating will be much better, but then you have to remember to switch the skin every time you want to play instead of spectating. 

It would be very nice to have a selector for which skin to use in spectator mode.

Another idea is to have a skin layer with spectator elements, but i think just having a separate skin would be better. 
For example, the one i did is just argon, but all of the elements are rescaled and moved for the sake of visibility, i cant imagine how i'd make this and the gameplay variant in the same skin with layers. Maybe hide the gameplay parts while editing the spectator parts? 

![screenshot](https://github.com/user-attachments/assets/f5dab05e-e7a5-433e-9ddd-3057417a8c26)

Either way, the main thing i want is the live text gone because we already have much better ways to display the names of players with skin elements, and the features above allow to have a clean default spectator layout

Ideally i would like to be able to edit the layout of the whole screen, i.e remove the leaderboard, add custom sprites etc. But that sounds more like tournament mode

**Comment by SupDos** [comment]
> i cant imagine how i'd make this and the gameplay variant in the same skin with layers

Using layers could definitely work, possibly by separating the HUD into two (gameplay and spectating), but it might need a better way of showing it
![image](https://github.com/user-attachments/assets/8e1d8e54-7d28-45f1-a850-7a8f0952d8a2)

then it would just switch between the two depending on whether you're in gameplay or spectating, same as it currently switches between different game mode HUDs depending on your game mode

If that's too complicated, another option could be to use a separate "Spectating" scene:
![image](https://github.com/user-attachments/assets/6bc84b0e-f6b4-4f4a-9b2f-f0b84e2c654b)

I feel like this method would be easier to understand for most users.

**Reply by spacekotek** [reply]
oh i forgot scenes exist, yeah i think it would be good as a scene
